test/simulation/install-autominer.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+CACHE_PATH=$1
+AUTOMINER_VENV_DIR=$2
+AUTOMINER_SOURCE=$3
+create_virtualenv () {
```

Comment:
This function should be called `install_autominer` or something like that because it does more than just creating a virtualenv